Resources/Prototypes/Entities/Mobs/NPCs/xeno.yml

```diff
@@ -4,6 +4,8 @@
   id: MobXeno
   parent: SimpleSpaceMobBase
   description: They mostly come at night. Mostly.
+  raffle:
+    settings: default
```

Comment:
it shouldn't be here.
raffles are a part of `GhostRole` component.

Reply:
just checked, theres already a line for raffles at 108-109

Reply:
Oh... and also thats xenos from the upstream.
Our content are located in `_CM14` folders.
Path to our xenos - `Resources/Prototypes/_CM14/Entities/Mobs/Xeno/`